recipes/bokeh/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "bokeh" %}
+{% set version = "0.11.1" %}
+{% set checksum = "34a73d8f68e7d6faaecc9f785cac8e9f76b88a7e7aacf66307076b03b4d3c472" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.5.2
+    - requests >=1.2.3
+    - pyyaml >=3.10
+    - python-dateutil >=2.1
+    - jinja2 >=2.7
+    - numpy >=1.7.1
+    - tornado >=4.3
+    - futures >=3.0.3         # [py2k]
+    - pandas
+    - matplotlib
+
+  run:
+    - python
+    - six >=1.5.2
+    - requests >=1.2.3
+    - pyyaml >=3.10
+    - python-dateutil >=2.1
+    - jinja2 >=2.7
+    - numpy >=1.7.1
+    - tornado >=4.3
+    - futures >=3.0.3         # [py2k]
+    - pandas
+    - matplotlib
+
+test:
+  imports:
+    - bokeh
+    - bokeh.application
+    - bokeh.application.handlers
+    - bokeh.charts
+    - bokeh.charts.builders
+    - bokeh.client
+    - bokeh.command
+    - bokeh.command.subcommands
+    - bokeh.core
+    - bokeh.core.compat
+    - bokeh.core.compat.mplexporter
+    - bokeh.core.compat.mplexporter.renderers
+    - bokeh.core.validation
+    - bokeh.models
+    - bokeh.models.widgets
+    - bokeh.plotting
+    - bokeh.sampledata
+    - bokeh.server
+    - bokeh.server.protocol
+    - bokeh.server.protocol.messages
+    - bokeh.server.views
+    - bokeh.sphinxext
+    - bokeh.themes
+    - bokeh.util
+
+about:
+  home: http://github.com/bokeh/bokeh
+  license: BSD 3-Clause
+  summary: Statistical and novel interactive HTML plots for Python
+
+extra:
+  recipe-maintainers:
+    - almarklein
```

Comment:
> I have more than enough on my plate I am happy to spectate here

Honestly, when asking devs to sign up as maintainers, this is sort of what I'm hoping for basically. The exact points that you have raised here about how to properly build the package are what I want to know. So, if I do something silly, someone who knows the codebase better can point it out to me. If either of you would be willing to that @bryevdv @damianavila, then I would be more than happy to add you here.